pkg/util/iptables/iptables_linux.go

```diff
@@ -81,7 +81,7 @@ func grabIptablesLocks(lockfilePath string) (iptablesLocker, error) {
 	}
 
 	// Roughly duplicate iptables 1.4.x xtables_lock() function.
-	if err := wait.PollImmediate(200*time.Millisecond, 2*time.Second, func() (bool, error) {
```

Comment:
I don't think extending this for testing purposes is correct. Doing something to avoid concurrent test runs might be preferable

Reply:
do we have a better solution? listening on the same unix socket in the test is already able to prevent concurrent test running.

Reply:
perhaps something test-specific, like locking on a test lock file in iptables#TestMain?

Reply:
The locking tests already override the iptables-1.6-compatible lock path. They should probably override the iptables-1.4-compatible lock path as well. (In the iptables 1.4 case the "path" is actually an abstract socket name (`"@xtables"`), but it can still just be overridden with an arbitrary alternate name.)